theano/gpuarray/c_code/dnn_conv_base.c

```diff
@@ -3,6 +3,43 @@ cudnnTensorDescriptor_t APPLY_SPECIFIC(input);
 cudnnTensorDescriptor_t APPLY_SPECIFIC(output);
 cudnnFilterDescriptor_t APPLY_SPECIFIC(kerns);
 
+static int c_check_groups_for_conv(cudnnConvolutionDescriptor_t desc, int groups) {
```

Comment:
I don't like the name. "Check" seem to indicate it would return True/False to me. What about renaming it to "get_groups_for_conv".